README.md

```diff
@@ -43,3 +43,15 @@ The ironic configuration can be overridden by various environment variables. The
 - OS_CONDUCTOR__INSPECT_TIMEOUT=1800 - timeout (seconds) for waiting for node inspection
 - OS_CONDUCTOR__CLEAN_CALLBACK_TIMEOUT=1800 - timeout (seconds) to wait for a callback from the ramdisk doing the cleaning
 - OS_PXE__BOOT_RETRY_TIMEOUT=1200 - timeout (seconds) to enable boot retries.
+
+Applying Patches to the image
+-----------------------------
+
+When building the image, it is possible to specify a patch of one or more
+upstream projects to apply to the image, passing a file with the patch list
+using the PATCH_LIST build argument.
+
```

Comment:
This might be obvious but should we say that the project needs to be hosted on opendev.org's gerrit?

Reply:
It's true at the moment, I will add a note on that